CONTRIBUTING.md

```diff
@@ -42,6 +42,11 @@ go build ./cmd/prometheus/
 make test         # Make sure all the tests pass before you commit and push :)
 ```
 
+To lint Go code changes do:
```

Comment:
Suggest expanding the word "lint" to a brief description to help people who might never have come across the tool before understand why this would be interesting to know.

Reply:
Fair, updated!